### PR TITLE
Add Quick Start to README, support for raw API requests, guess input file type for run

### PR DESCRIPTION
**Pull request by kemp-google**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,18 +4,43 @@
...
+3. Inside the Cloud Shell, run the command
 
-Once Go is installed, you can build the tools by running:
+        go get github.com/googlegenomics/pipelines-tools/...
 
-```
-$ go get github.com/googlegenomics/pipelines-tools/...
-```
+   This command downloads and installs the pipelines tools.  Note that to build
+   these tools the Cloud Shell you will need the [Go][2] tool chain.
```
Comment (gkelly) [inline README.md]: "the tools the" -> "these tools in the"

File: README.md [hunk 2]
```diff
@@ -4,18 +4,43 @@
...
+
+        pipelines run --inputs=${BUCKET}/input --outputs=${BUCKET}/output sha1.script
+
+8. Check the generated output file:
+
+        gsutil cat ${BUCKET}/output
 
-This will produce binaries in $GOPATH/bin for each tool.
+That's it: you've run your first pipeline.  For more information about the
+scripting language supported by the pipelines tool, check out the [source
```
Comment (gkelly) [inline README.md]: "scripting language" -> "input format"

File: README.md (hunk 1 above)
Comment (kemp-google) [reply README.md]: -> outside the

File: README.md (hunk 2 above)
Comment (kemp-google) [reply README.md]: done

**Comment by kemp-google** [comment]
PTAL

**Pull request merged**